### ISSUE TITLE
Extension

### ISSUE THREAD

**Issue by kokolo12325** [Alex313031/quark-player#35]
Is it possible to insert additional extensions besides adblocker?

**Comment by kokolo12325** [comment]
Thank You

**Comment by Alex313031** [comment]
@kokolo12325 @DI555 No, because almost all of the services use Widevine and proprietary video players, i.e. it isn't a normal simple <video> tag that uses Chromium's built in video player. Widevine prevents piracy by encrypting the media stream, and preventing access to the video by anything other than Chromium's main process. Trying to inject .js with an extension to change attributes of the playing video would thus not work. Exceptions to this are YouTube and Woolyss, but it's not worth the amount of development effort to me to make something like that work on just those two services.
